api/tests/channels/test_escalation_hook.py

```
@@ -155,3 +155,47 @@ async def test_routing_reason_defaults_none_when_missing(self):
+
+
+@pytest.mark.unit
+@pytest.mark.asyncio
+async def test_escalation_hook_creates_escalation_for_queue_medium(
+    sample_incoming_message,
+    mock_rag_service,
+):
+    from app.channels.gateway import ChannelGateway
+    from app.channels.hooks.escalation_hook import EscalationPostHook
+
+    async def rag_query(*args, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `args`

(ARG001)